css/navbar.css

```diff
@@ -76,15 +79,15 @@
   color: var(--night);
 }
 
-.fa-cart-shopping,
-.fa-heart {
+.nav-cart-icon,
+.nav-heart-icon {
   font-size: 3rem !important;
```

Comment:
remove important and use specificity to apply styling

Reply:
thanks for pointing it out.

Reply:
i fixed this.